src/components/containers/PanelHeader.js

```diff
@@ -63,19 +62,24 @@ class PanelHeader extends Component {
                 label={addAction.label}
               />
               {this.state.addPanelOpen && (
-                <ModalBox onClose={this.togglePanel} relative>
-                  {addAction.handler.map(({label, handler}) => (
-                    <p
-                      key={label}
-                      onClick={() => {
-                        handler(this.context);
-                        this.togglePanel();
-                      }}
-                    >
-                      {label}
-                    </p>
-                  ))}
-                </ModalBox>
+                <div className="Select">
```

Comment:
Please follow BEM here :)

Reply:
OK, these classes were reused from react-select component, which doesn't use BEM obviously. I think we benefit from reusing them, so that this dropdown always ends up looking like a dropdown of all combo boxes. But if BEM is more important, I could copy paste them with appropriate names...

Reply:
@VeraZab what do you think? I think it makes sense to copy-paste them that way we maintain control over their look no matter what happens when we upgrade `react-select`

Reply:
I'd be in favor of using select classes as it makes it more obvious that it's not our own css, maybe add a comment?

And maybe that `_dropdown.scss` stylesheet could be used for this component also, we want the same overrides to apply right?

so maybe we could just add the `.dropdown` class on the enclosing div to have these style overrides apply. I think it's ok, we're dealing with a dropdown here too right